packages/editor-ui/src/stores/templates.store.ts

```diff
@@ -135,11 +140,13 @@ export const useTemplatesStore = defineStore(STORES.TEMPLATES, {
 		 */
 		getWebsiteCategoryURL() {
 			return (id: string) => {
-				return `${TEMPLATES_URLS.BASE_WEBSITE_URL}/?categories=${id}&${TEMPLATES_URLS.UTM_QUERY}&utm_instance=${this.getCurrentN8nPath}`;
+				return `${TEMPLATES_URLS.BASE_WEBSITE_URL}/?categories=${id}&${
+					TEMPLATES_URLS.UTM_QUERY
+				}&utm_instance=${this.getCurrentN8nPath}&utm_n8n_version=${useRootStore().versionCli}`;
 			};
 		},
 		getCurrentN8nPath(): string {
-			return `${window.location.host}${window.BASE_PATH}`;
+			return `${window.location.protocol}//${window.location.host}${window.BASE_PATH}`;
 		},
```

Comment:
Since this never changes, does this still need to be a `getter`? could we not move this to `state` so that it's evaluated only once?

Reply:
Yeah, makes sense